addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -105,8 +105,16 @@ var SnippetEditor = Widget.extend({
         }
 
         var _animationsCount = 0;
-        var postAnimationCover = _.throttle(() => this.cover(), 100);
-        this.$target.on('transitionstart.snippet_editor, animationstart.snippet_editor', () => {
+        var postAnimationCover = _.throttle(() => {
```

Comment:
In fact, does this need to be throttled? If you hide the overlay during animations, etc.

Reply:
Indeed, I removed throttle.